### ISSUE TITLE
Tests fail for me locally

### ISSUE THREAD

**Issue by robertzk** [avantcredit/r-servicetests#3]

**Comment by khalidharun** [comment]
Which test failed? Mine passed using the new repo.

**Comment by robertzk** [comment]
@khalidharun I see:

```

1. Error: clear_registry works -------------------------------------------------
cannot change value of locked binding for 'env'
1: withCallingHandlers(eval(code, new_test_environment), error = capture_calls)
2: eval(code, new_test_environment)
3: eval(expr, envir, enclos)
4: register_service_call(fn1(1, 2)) at test-servicetests.R:10

```

**Comment by robertzk** [comment]
All of them.

**Comment by khalidharun** [comment]
ok, I think I got this one

**Comment by khalidharun** [comment]
fixed in #4
